src/main/java/org/codehaus/mojo/license/CheckProjectLicenseMojo.java

```diff
@@ -0,0 +1,100 @@
+package org.codehaus.mojo.license;
+
+/*
+ * #%L
+ * License Maven Plugin
+ * %%
+ * Copyright (C) 2008 - 2012 CodeLutin, Codehaus, Tony Chemit
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+
+import javax.inject.Inject;
+
+import java.io.File;
+
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProjectHelper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The goal to check if there are license files in project.
+ *
+ * @author swalendzik
+ * @since 2.7
+ */
+@Mojo(name = "check-project-license", threadSafe = true)
+public class CheckProjectLicenseMojo extends AbstractLicenseNameMojo {
+    private static final Logger LOG = LoggerFactory.getLogger(CheckProjectLicenseMojo.class);
+
+    // ----------------------------------------------------------------------
+    // Mojo Parameters
+    // ----------------------------------------------------------------------
+
+    /**
+     * Project license file to check.
+     *
+     * @required
+     * @since 2.7
+     */
+    @Parameter(property = "license.licenceFile", defaultValue = "${basedir}/LICENSE.txt")
```

Comment:
`File` parameters are always aligned by current project, so `${basedir}` is not needed.